ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -357,6 +366,11 @@ func convertToContainerDef(context *project.Context, inputCfg *config.ServiceCon
 		outputContDef.LinuxParameters.SetSharedMemorySize(shmSize)
 	}
 
+	if inputCfg.Tmpfs != nil {
```

Comment:
what's this check doing? A comment explaining the same would be awesome.

Reply:
I figured we'd only set this if the value specified in the `docker-compose.yml`. Since `convertToTmpfs` returns a slice rather than a pointer to a slice, I did the null check here to avoid setting tmpfs to be an empty slice and therefore potentially interfere with any default behavior docker has for tmpfs being null. Just to verify, is that something we should account for?

Reply:
I also think that a short comment would improve the ease of readability for the code.

Reply:
i see. slices have a nice property in go that len() will return 0 both for null and empty slices. So, that might make it more explicit `if len(inputCfg.Tmpfs) == 0`.